setup.py

```diff
@@ -115,30 +116,40 @@ def user_dir():
 
 class DevelopCmd(develop):
     prefix_targets = [
-        ("nbconvert/templates", 'jdaviz-default'),
-        ("voila/templates", 'jdaviz-default')
+        (os.path.join("nbconvert", "templates"), 'jdaviz-default'),
+        (os.path.join("voila", "templates"), 'jdaviz-default')
     ]
 
     def run(self):
         target_dir = os.path.join(sys.prefix, 'share', 'jupyter')
         if '--user' in sys.prefix:  # TODO: is there a better way to find out?
             target_dir = user_dir()
         target_dir = os.path.join(target_dir)
+        is_win = sys.platform.startswith('win')
 
         for prefix_target, name in self.prefix_targets:
             source = os.path.join('share', 'jupyter', prefix_target, name)
             target = os.path.join(target_dir, prefix_target, name)
             target_subdir = os.path.dirname(target)
             if not os.path.exists(target_subdir):
                 os.makedirs(target_subdir)
```

Comment:
pathlib.mkdir is a no-op automatically if the path exists, no need to check

```suggestion
            target_subdir.mkdir(parents=True, exist_ok=True)
```